pkgs/applications/networking/browsers/firefox/wrapper.nix

```diff
@@ -1,5 +1,6 @@
 { stdenv, lib, makeDesktopItem, makeWrapper, lndir, config
-, replace, fetchurl, zip, unzip, jq, xdg_utils
+
```

Comment:
not sure why this empty line is still there, but thats nitpicking :D

Reply:
![image](https://user-images.githubusercontent.com/22085373/102271190-58b93180-3f1f-11eb-9663-08be24929a20.png)